packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -74,7 +75,25 @@ export const activatePlugin: StepHandler<ActivatePluginStep> = async (
 			throw new Exception( 'Unable to activate plugin' );
 		`,
 	});
-	if (result.text !== 'Plugin activated successfully') {
+
+	/**
+	 * A valid plugin activation response is either
+	 * a 200 status code response with the text 'Plugin activated successfully'
+	 * or a redirect status code (300-399) response with the text ''
+	 *
+	 * Some plugins redirect on activation.
```

Comment:
How do we know it's not an error-related redirection?

Reply:
WP-CLI checks `is_wp_error` https://github.com/wp-cli/extension-command/blob/main/src/Plugin_Command.php#L384

I updated my code to also cover scenarios when plugins create output. It contains a explanation why I think we shouldn't rely on the `is_wp_error` check https://github.com/WordPress/wordpress-playground/pull/2066/files#diff-4a02b123bb6311fff3ec089ca1ade75c75f1af35aaa4f1a717f14d444fc9b485R81-R84.